lagunita/lms/static/css/vendor/su-identity.css

```diff
@@ -1,291 +1,231 @@
+/*
```

Comment:
This comment doesn't add clarity; let's drop it.

And as a side note:
When code _does_ warrant comments, please don't use the long string of `=`s.
Those show up as false positives when grepping for conflict markers.